install/install_openadapt.ps1

```diff
@@ -70,205 +75,181 @@ function CheckCMDExists {
             exit
         }
 
-        Write-Host "Installing git, click 'Yes' if prompted for permission"
+        Write-Host "Installing git..."
         Start-Process -FilePath $gitInstaller -Verb runAs -ArgumentList '/VERYSILENT /NORESTART /NOCANCEL /SP- /CLOSEAPPLICATIONS /RESTARTAPPLICATIONS /COMPONENTS="icons,ext\reg\shellhere,assoc,assoc_sh"' -Wait
         Remove-Item $gitInstaller
 
-        #Refresh Path Environment Variable
+        # Refresh Path Environment Variable
         $env:Path = [System.Environment]::GetEnvironmentVariable("Path", "Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path", "User")
 
         # Make sure git is now available
-        $gitExists = CheckCMDExists($gitCmd)
+        $gitExists = CheckCMDExists $gitCmd
         if (!$gitExists) {
-            Write-Host "Error after installing git. Uninstalling, click 'Yes' if prompted for permission"
+            Write-Host "Error after installing git. Uninstalling..."
             Start-Process -FilePath $gitUninstaller -Verb runAs -ArgumentList '/VERYSILENT', '/SUPPRESSMSGBOXES', '/NORESTART' -Wait
+            Cleanup
             exit
         }
     }
     # Return the git command
     return $gitCmd
 }
-
-
-function GetVSCppRedistCMD {
-    # Check if Visual C++ Redist is installed
-    $ErrorActionPreference = "SilentlyContinue"
-    $vcredistExists = Get-ItemPropertyValue -Path $VCRedistRegPath -erroraction 'silentlycontinue' -Name Installed
-    $ErrorActionPreference = "Continue"
-
-    if (!$vcredistExists) {
-        # Install Visual C++ Redist
-        Write-Host "Downloading Visual C++ Redist"
-        $ProgressPreference = 'SilentlyContinue'
-        Invoke-WebRequest -Uri $VCRedistInstallerLoc -OutFile $VCRedistInstaller
-        $exists = Test-Path -Path $VCRedistInstaller -PathType Leaf
-        if (!$exists) {
-            Write-Host "Failed to download Visual C++ installer"
-            Cleanup
-            exit
-        }
-
-        Write-Host "Installing Visual C++ Redist, click 'Yes' if prompted for permission"
-        Start-Process -FilePath $VCRedistInstaller -Verb runAs -ArgumentList "/install /q /norestart" -Wait
-        Remove-Item $VCRedistInstaller
-
-        if ($LastExitCode) {
-            Write-Host "Failed to install Visual C++ Redist: $LastExitCode"
-            Cleanup
-            exit
-        }
-    }
-}
-
 ################################   FUNCTIONS    ################################
 
 
 ################################   SCRIPT    ################################
 
-# Check and Install TesseractOCR -> Python 3.10 -> Git -> VS C++ Redist.
+# Create a new directory and run the setup from there
+New-Item -ItemType Directory -Path $setupdir -Force > $null
+Set-Location -Path $setupdir > $null
+
+# Check and Install the required softwares for OpenAdapt
 $tesseract = GetTesseractCMD
-RunAndCheck "$tesseract --version" "check tesseract version"
+RunAndCheck "$tesseract --version" "check TesseractOCR" > $null
```

Comment:
The inout is being redirected to null as I do not want to clutter the terminal with the poetry installation logs. In my opinion, user point of view, it is not necessary to show the logging of all the installation of libraries. It will make sense to remove unnecessary logs like installtion etc. 

I hope this make sense.

Reply:
@KrishPatel13 let's remove the redirect, if the installation fails it will be difficult to debug without logs.

Reply:
Addressed in: [cfb69ab](https://github.com/MLDSAI/OpenAdapt/pull/312/commits/cfb69abb1c2de8b8585014b05f42b403a9b84f05)